dashboard/config/initializers/recaptcha.rb

```diff
@@ -0,0 +1,4 @@
+Recaptcha.configure do |config|
+  config.site_key = CDO.recaptcha_site_key
```

Comment:
Please make sure to add the keys in Chef for all environments. Is there anything required for dev environments? 

What happens if this key is somehow not set?